### PR TITLE
Waterfox G6.0.3

### PR DESCRIPTION
**Pull request by aaronliu0130**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121867)

### CODE REVIEW

**Comment by aaronliu0130** [comment]
Excuse me?

**Comment by aaronliu0130** [comment]
Is there a way to resolve the thing where installing without a scope fails? I’ve tried adding a scope field to the root

**Comment by stephengillie** [comment]
> Is there a way to resolve the thing where installing without a scope fails? I’ve tried adding a scope field to the root

This would be up to the package's developer. Unfortunately, this package's behavior is somewhat unique - I can't think of another package which uses elevation to determine install scope. Most use a switch, and some will only install in one or the other. 

If it helps, the Machine scope is passing. User scope is failing Manual Validation due to the dependency not having a user scope:

```
2023-10-04 14:01:08.300 [CLI ] Starting installer selection.
2023-10-04 14:01:08.300 [CLI ] Installer [X64,burn,Machine,] not applicable: Installer scope does not match required scope: Machine != User
2023-10-04 14:01:08.303 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.32919.0
2023-10-04 14:01:08.304 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:11a
```

**Comment by aaronliu0130** [comment]
I'm pretty sure Firefox's offline installer (e.g. https://cdn.stubdownloader.services.mozilla.com/builds/firefox-latest-ssl/en-US/win/dd343b8ca7f4c9e3bea506184bbaaa626e70c72547a50f8dbe993bd891819693/Firefox%20Setup%20118.0.1.exe) also does that

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml
Comment (MrAlex94) [inline manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml:27]: Waterfox doesn't require the Visual C++ redist files as they come bundled already.

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (MrAlex94) [inline manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:7]: This should be BrowserWorks

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (MrAlex94) [inline manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:16]: As above, should be BrowserWorks.

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (MrAlex94) [inline manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:18]: Probably best to update this to:
> Waterfox is an independent, open-source web browser built for speed, privacy, and user customization.

**Review by MrAlex94 — CHANGES_REQUESTED** [review]
Have added some recommendations. Since VCRedist isn't needed, may stop the issues?

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml
Comment (aaronliu0130) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml:27]: This was the result of another failure in #121246 . Did something change between 6.0.2 and 6.0.3?

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (aaronliu0130) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:7]: Unless the publisher is also listed as BrowserWorks in the control panel, this will not be changed.

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (aaronliu0130) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:18]: How about performance, privacy and customization? I don’t see much reason to change the rest

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (aaronliu0130) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:16]: At least last time I checked this was still what it said on the website

Comment (MrAlex94) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:16]: Website and browser both list BrowserWorks:
image
image

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml
Comment (MrAlex94) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:18]: The rest is important as Waterfox Limited is no longer the legal entity Waterfox belongs to, BrowserWorks is.

Comment (aaronliu0130) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.locale.en-US.yaml:18]: The short description does not reference any legal entities.

File: manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml
Comment (MrAlex94) [reply manifests/w/Waterfox/Waterfox/G6.0.3/Waterfox.Waterfox.installer.yaml:27]: Yes, I added the missing redist files. Didn't realise they were missing 😅

**Comment by aaronliu0130** [comment]
I wish installation logs were public and we didn't need maintainers for them

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335146 

@aaronliu0130, 

The logs are publicly available. Inside the ADO run, there's a link to the run [artifact zip file](https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_apis/build/builds/504001/artifacts?artifactName=InstallationVerificationLogs&api-version=7.1&%24format=zip). Or maybe run the [scripted process](https://github.com/microsoft/winget-pkgs/blob/27318841518093cca2785835b155df661d8b5950/Tools/ManualValidationPipeline.ps1#L992) with the [run link](https://dev.azure.com/ms/ed6a5dfa-6e7f-413b-842c-8305dd9e89e6/_build/results?buildId=504001) in your clipboard.

**Comment by stephengillie** [comment]
> Unresolved conversations
 - 2 conversations must be resolved before merging.

**Comment by aaronliu0130** [comment]
I did not resolve to await a reply at first, but it doesn't appear any more action is needed so they are now resolved..

**Review by stephengillie — APPROVED** [review]

**Pull request merged**
